### PR TITLE
2 bugfixes: always "1" output, nolock=yes and slave=yes combination

### PR DESCRIPTION
**Pull request by michaeldg**

### CODE REVIEW

File: bgbackup.sh [hunk 1]
```diff
@@ -128,6 +129,7 @@ function backer_upper {
         mysql -u "$backupuser" -p"$backuppass" -e "SET GLOBAL wsrep_desync=ON;"
     fi
     log_info "Beginning ${butype} Backup"
+    log_info "Executing Innobackupex comamnd: $innocommand"
```
Comment (bstillman) [inline bgbackup.sh]: you misspelled "command"

File: bgbackup.sh [hunk 2]
```diff
@@ -502,7 +504,7 @@ fi
 mysqlcreate
 
 # Check that mysql client can connect
-$mysqlcommand "SELECT 1 FROM DUAL"
+$mysqlcommand "SELECT 1 FROM DUAL" 1>/dev/null
```
Comment (bstillman) [inline bgbackup.sh:507]: Did you test this both with correct info and incorrect to verify it works both ways as expected?

Comment (michaeldg) [reply bgbackup.sh:507]: Looking at the code it just looks at the exit code. I do not think such test is needed for fixing this bug.

File: bgbackup.sh (hunk 1 above)
Comment (michaeldg) [reply bgbackup.sh]: Ty, will fix it

File: bgbackup.sh (hunk 2 above)
Comment (bstillman) [reply bgbackup.sh:507]: Look at line 508 just below. It's looking for that to equal 1. If it does, then it should be returning the error on line 509. To determine if it's exiting with a 1 (line 512) or just displaying what it is selecting, change it to "select 0 from dual" and test.

Comment (michaeldg) [reply bgbackup.sh:507]: I just change what the command outputs, I did not change it's exit code. Output redirection does not have to do with exit codes. The $? checks on the exit code, not the output.

Comment (bstillman) [reply bgbackup.sh:507]: Is it exiting with 1, or is it just outputting from the query?

**Pull request merged**